### DISCUSSION TITLE
Wayvnc + sway in docker?

### DISCUSSION THREAD

**Discussion by RajaMuneerBaigal** [any1/wayvnc#165] [General]
I am currently working with docker. I have a base alpine image and have installed swaywm on top of it. According to swawm/sway mainter it is not possible to run sway in docker container. somehow I got away running it with few errors. Now I want to add vnc to access my setup environment remotely through a vnc viewer. But I am not sure if its possible or not ? If its possible can anyone let me know how to use wayvnc in a docker image. Any help would be appreciated

**Comment by lack** [comment]
There shouldn't be anything special about wayvnc in docker?
- wayvnc needs to find the Wayland Unix domain socket that sway opened (usually $XDG_RUNTIME_DIR/wayland-1) so you will need to configure sway to put that somewhere that wayvnc can find it (probably by setting $XDG_RUNTIME_DIR for both sway and wayvnc)
- pass the VNC port (5900 by default) so you can get in :)

Maybe you'll need something additional if you want to use wayvnc's DRM support?
